modules/dnn/src/layers/recurrent_layers.cpp

```diff
@@ -266,7 +266,7 @@ class LSTMLayerImpl CV_FINAL : public LSTMLayer
                          std::vector<MatShape> &internals) const CV_OVERRIDE
     {
         CV_Assert((!usePeephole && blobs.size() == 5) || (usePeephole && blobs.size() == 8));
-        CV_Assert(inputs.size() == 1);
+        CV_Assert(inputs.size() <= 3);
```

Comment:
Better to check that number of inputs is 1 or 3.

Reply:
Done!